api/openapi-spec/README.md

```diff
@@ -21,11 +21,11 @@ For example:
     "/api/v1/namespaces/{namespace}/pods/{name}": {
         ...
         "get": {
-        ...
+            ...
```

Comment:
use empty spaces, instead of Tab?

Reply:
@dixudx This PR adds empty spaces instead of Tab, and the existing indents also are written with spaces.